LongestPalindrome.java

```diff
@@ -0,0 +1,22 @@
+cilass LongestPalindrome {
+    // Space Complexity - O(N) 
```

Comment:
Space will be O(1) . In worst max characters in set can be all valid ascii characters i.e. 256 which is constant